packages/tokens/config.js

```diff
@@ -0,0 +1,66 @@
+// eslint-disable-next-line no-undef
+const StyleDictionary = require("style-dictionary");
```

Comment:
error was "`require` is not defined" which I think is expected here / fine to ignore?

Reply:
Hmmm, I think this isn't expected. Looks like eslint doesn't know that this file is in a node environment.

Our eslintrc is trying to set that up, though, which we can see in https://github.com/chanzuckerberg/lp-design-system/blob/925402f0b91581f3fdaf1078b823b9b6b43eaa10/.eslintrc#L45-L51.

Not sure why it doesn't seem to be working. But we'll want to dig into why that isn't working as expected.

Reply:
I think instead of that you should make its env `node` this is probably defined in the top level eslint, we can make an override for some files.

Reply:
@ahuth beat me to it

Reply:
good call, thanks :)